cockatrice/src/window_sets.cpp

```diff
@@ -173,6 +193,31 @@ void WndSets::actRestore()
     close();
 }
 
+void WndSets::actResetSort()
+{
+    view->header()->setSortIndicator(-1, Qt::DescendingOrder);
```

Comment:
A minor one: you could use a define a explicit name for this magic number, eg.
```
#define SORT_RESET -1
```
and use the name instead of the value.